OpenEXR/IlmImf/ImfTiledInputFile.cpp

```diff
@@ -745,7 +745,10 @@ TiledInputFile::TiledInputFile (const char fileName[], int numThreads):
 
             delete _data->_streamData;
         }
-
+        if (_data)
```

Comment:
these null checks aren't necessary, and there's a few in a row that check != 0 or the pointer. Be better to just squash them all to delete....